### PR TITLE
Dev design

### PR DESCRIPTION
**Pull request by sylvainar**
V2.0 incoming !

- Installed Bootstrap & jQuery
- Full responsive design
- Switched map to OSM instead of Mapbox
- WikiJourney's API is now loading in asynchronous
- Setting new logos
- CSS and Javascript refactoring
- Increase pagespeed 
- Give the possibility to choose the language for the request

### CODE REVIEW

File: style/design.css [hunk 1]
```diff
@@ -41,49 +53,63 @@ h1, h2, h3, h4 {
...
-	float: left;
-	width: 120px;
+.vcenter {
+    display: inline-block;
+    vertical-align: middle;
+    float: none;
+}
+
+.shadowed{
+	-webkit-box-shadow: 0 3px 6px 0 #999;
```
Comment (tikiki) [inline style/design.css]: The prefixes might not be needed

File: style/design.css [hunk 2]
```diff
@@ -41,49 +53,63 @@ h1, h2, h3, h4 {
...
-	margin: auto;
 }
 
-#languagesBox a {
-	color: white;
+.stores_logos_block img {
+	height: 40px;
+	display: inline-block;
+	margin: 5px;
+	-webkit-box-shadow: 0 12px 6px -6px #999;
```
Comment (tikiki) [inline style/design.css]: Same, no prefixes needed

File: style/design.css [hunk 3]
```diff
@@ -111,229 +137,95 @@ label {
...
+	height:25px !important;
+	margin-right: 10px;
 }
 
-.POPUP_title {
-	font-weight: bold;
-	margin: auto;
-	text-align: center;
+.notshrink{
+	-webkit-filter: drop-shadow(2px 5px 3px #999);
```
Comment (tikiki) [inline style/design.css]: Why a filter instead of a normal drop-shadow?

File: style/design.css [hunk 4]
```diff
@@ -111,229 +137,95 @@ label {
...
+	
 }
 
-.POPUP_img {
-	max-height: 100px;
-	width: 100px;
-	margin: auto;
-	display: block;
+.logoNavbar{
+	-webkit-transition: 0.4s ease;
```
Comment (tikiki) [inline style/design.css]: No prefix needed

File: style/design.css [hunk 5]
```diff
@@ -111,229 +137,95 @@ label {
...
-height:8px;
-background-color:#eeeeee;
+.footer {
+	padding-top: 10px;
+	padding-bottom: 10px;
+	font-size: 14px;
+	background-color: rgba(137, 160, 173, 0.5);
+	
+	box-shadow: inset 0 8px 6px -6px #999;
+	-webkit-box-shadow: inset 0 8px 6px -6px #999;
```
Comment (tikiki) [inline style/design.css]: No prefix needed

File: style/map.css [hunk 6]
```diff
@@ -0,0 +1,327 @@
...
+	width: 200px;
+	margin: 30px auto;
+}
+
+.drawer {
+	position: absolute;
+	background-color: white;
+	
+
+	-webkit-box-shadow: 20px 12px 6px -6px #999;
```
Comment (tikiki) [inline style/map.css]: No prefix needed

File: style/map.css [hunk 7]
```diff
@@ -0,0 +1,327 @@
...
+	-moz-box-shadow: 20px 12px 6px -6px #999;
+	box-shadow: 3px 0 8px 0 #999;
+
+	-webkit-transition: 0.4s ease;
```
Comment (tikiki) [inline style/map.css]: Same, no prefix

File: style/map.css [hunk 8]
```diff
@@ -0,0 +1,327 @@
...
+	height: 100%;
+}
+
+.drawer-bottom{
+	width: 100%;
+	bottom: -50px; /* Hidden in the beginning, JS will show it */
+}
+
+#POI_CART_BLOCK{
+	z-index: 6000;
```
Comment (tikiki) [inline style/map.css]: This is not OVER 9000!!!

File: style/map.css [hunk 9]
```diff
@@ -0,0 +1,327 @@
...
+.eltCart{
+	overflow-y: hidden;	/* To cut the background */
+	position: relative;
+}
+
+.eltCartBlur{
+	position: absolute;
+
+	width: 100%;
+	height: 200px; /* "big" value to fullfill the element */
```
Comment (tikiki) [inline style/map.css]: fullfill??

File: style/design.css (hunk 3 above)
Comment (sylvainar) [inline style/design.css]: Because the logo is round so the shadow fits it

File: style/map.css [hunk 10]
```diff
@@ -0,0 +1,327 @@
...
+
+	/* The background url is given inline in JS */
+	background-size: cover;
+	background-position: center; 
+
+	/* Fallback to SVG filters */
+	filter: url('../images/design/blur.svg#blur');
+	/* Normal filter */	
+	-webkit-filter: blur(2px);
```
Comment (tikiki) [inline style/map.css]: Only the webkit prefix is needed (-o- is dead since quite some time...)

File: style/map.css (hunk 9 above)
Comment (sylvainar) [inline style/map.css]: Yeay, it's a bloc behind another, if I don't set the height it collapse to 0px.

File: style/design.css [hunk 11]
```diff
@@ -111,229 +135,126 @@ label {
...
 }
 
-/* ===> Footer <=== */
-
-.footer {
-	border-top: 1px solid black;
-	padding: 20px;
-	font-size: 14px;
+.notshrink{
+	filter: drop-shadow(2px 5px 3px #999);
```
Comment (tikiki) [inline style/design.css]: I don't know why the -webkit- prefix disappeared here, I never said it wasn't necessary...
(Using an autoprefixer would avoid this kind of problem)

**Pull request merged**